src/index.html

```diff
@@ -24,10 +24,9 @@
       <hr />
       <div class="container-footer">
         <p class="credits">
-          © 2022 Key. All Right Reserved | Created by
+         MovieApp © 2022 Key. All Right Reserved | Created by
```

Comment:
I think we can use the created under CC license instead so others can use our code.